frontend/src/components/ButtonSpacer/ButtonSpacer.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+
+const ButtonSpacer = () => 
+    {
+        return(
+            <div>
+            &nbsp;&nbsp;&nbsp;
+        </div>
+        )
+    };
+
+
+export default ButtonSpacer
```

Comment:
Im not sure that we need one component to add some whitespace. I think the [<br>](https://www.w3schools.com/tags/tag_br.asp) tag might do the same thing as this component.

Reply:
I deleted the component and just used the tag instead. Thank you!

Reply:
Im not sure why it didnt include my full text but the <br> tag will add a new line if you want to put that wherever the ButtonSpacer tag exists. Or maybe im misunderstanding what you are trying to do. Would you like to add whitespace in the button?

Reply:
No, you were right. Looking back, I am not sure why I made that component. It was only meant to add some space between buttons, not in the button.

Reply:
I think that if I put html in the comment it doesnt add it for some reason. Thats weird. Really what ive been trying o refer to is https://www.w3schools.com/tags/tag_br.asp the br tag.